nearlib/package-lock.json

```diff
@@ -32,6 +32,15 @@
         }
       }
     },
+    "JSONStream": {
```

Comment:
hmhmhm, do we really need package-lock.json file?

Reply:
it's common practice to avoid accidental install of different dependency versions (and so irreproducible builds). I think it's fine, dependencies don't change that often and it's always possible to resolve conflicts by removing file and running `npm install`